src/OrbitGl/PrimitiveAssembler.h

```diff
@@ -51,8 +38,8 @@ AddInternalMethods, ResetElements(), GetLayers() and DrawLayers().
 **/
 class PrimitiveAssembler {
  public:
-  explicit PrimitiveAssembler(BatcherId batcher_id, PickingManager* picking_manager = nullptr)
-      : batcher_id_(batcher_id), picking_manager_(picking_manager) {
+  explicit PrimitiveAssembler(Batcher* batcher, PickingManager* picking_manager = nullptr)
+      : batcher_(batcher), picking_manager_(picking_manager) {
```

Comment:
It might make sense to add this to the constructor:
```
ORBIT_CHECK(batcher_ != nullptr);
ORBIT_CHECK(picking_manager_ != nullptr);
```

Reply:
Added, also erase the default value for picking_manager.

Reply:
I reverted this. Only added checker Batcher as a nullptr but for now not PickingManager. It's possible to use a PrimitiveAssembler without picking_manager if you don't use pickable methods.